FirebaseDataConnect/Tests/Integration/KitchenSinkOperations.swift

```diff
@@ -0,0 +1,864 @@
+import FirebaseDataConnect
+import Foundation
+
+// MARK: Connector Client Extension
+
+public extension KitchenSinkClient {
+  func createTestIdMutationRef(id: UUID) -> MutationRef<
+    CreateTestIdMutation.Data,
+    CreateTestIdMutation.Variables
+  > {
+    var variables = CreateTestIdMutation.Variables(id: id)
+
+    let request = MutationRequest(operationName: "createTestId", variables: variables)
+    let ref = dataConnect.mutation(
+      request: request,
+      resultsDataType: CreateTestIdMutation.Data.self
+    )
+    return ref as! MutationRef<CreateTestIdMutation.Data, CreateTestIdMutation.Variables>
+  }
+
+  func createTestAutoIdMutationRef(
+  ) -> MutationRef<CreateTestAutoIdMutation.Data, CreateTestAutoIdMutation.Variables> {
+    var variables = CreateTestAutoIdMutation.Variables()
+
+    let request = MutationRequest(operationName: "createTestAutoId", variables: variables)
+    let ref = dataConnect.mutation(
+      request: request,
+      resultsDataType: CreateTestAutoIdMutation.Data.self
+    )
+    return ref as! MutationRef<
+      CreateTestAutoIdMutation.Data,
+      CreateTestAutoIdMutation.Variables
+    >
+  }
+
+  func createStandardScalarMutationRef(id: UUID,
+
+                                       number: Int,
+
+                                       text: String,
+
+                                       decimal: Double)
+    -> MutationRef<CreateStandardScalarMutation.Data,
+      CreateStandardScalarMutation.Variables> {
+    var variables = CreateStandardScalarMutation.Variables(
+      id: id,
+      number: number,
+      text: text,
+      decimal: decimal
+    )
+
+    let request = MutationRequest(operationName: "createStandardScalar", variables: variables)
+    let ref = dataConnect.mutation(
+      request: request,
+      resultsDataType: CreateStandardScalarMutation.Data.self
+    )
+    return ref as! MutationRef<
+      CreateStandardScalarMutation.Data,
+      CreateStandardScalarMutation.Variables
+    >
+  }
+
+  func createScalarBoundaryMutationRef(id: UUID,
+
+                                       maxNumber: Int,
```

Comment:
nit: Many extra blank lines in this file.

Reply:
This is generated code. I ran it via our style script and this is the cleaned up version.

Reply:
Will need to work on fixing the generation to reduce blank lines. Some of these won't removed by swift formatting.